guidelines/REUSE.md

```diff
@@ -1,85 +1,100 @@
 # Reuse Philosophy
+
 We encourage reuse and patterns through references.
 
 ## What is reusable
+
 The following types are reusable, as defined by the spec:
 
-* Operations
 * Parameters
-* Responses
 * Models (or Schema Objects in general)
+* Responses
+* Operations (_Operations can only be remote references_)
 
 ## Reuse strategy
-When reusing components in an API design, a pointer is created from the definition to target design.  The references are maintained in the structure, and can be updated by modifying the source definitions.  This is different from a "copy on design" approach where references are injected into the design itself.
 
-The reuse technique is transparent between JSON or YAML and is lossless when converting between the two.
+When reusing components in an API design, a reference is created from the definition to target entity. The references are maintained in the structure, and can be updated by modifying the source definitions. This is different from a "copy on design" approach where references are injected into the design itself.
```

Comment:
Sorry for the late reply. Just read the comments on this section and there are two issues brought up that I'm not sure I agree with. I've had a quick chat with @whitlockjc directly (sorry @jasonh-n-austin), and think there may be room for discussion. Note I'm adding this comment after reading the comments above but before reading the modified doc. The two issues are:
- There's no difference between JSON References and YAML Anchors, and there's no difference between 'injecting' and 'referencing' (I believe there's a difference in both cases).
- Local JSON References act differently in the OpenAPI spec (they don't).

Since it's late for today, I hope we can continue the discussion tomorrow. I'd rather not dive into the details now to not pull in people to a discussion at this hour.